src/test/java/org/folio/rest/impl/SearchOrderLinesTest.java

```diff
@@ -0,0 +1,85 @@
+package org.folio.rest.impl;
+
+import static org.folio.rest.RestVerticle.OKAPI_HEADER_TENANT;
+import static org.folio.rest.utils.TenantApiTestUtil.deleteTenant;
+import static org.folio.rest.utils.TenantApiTestUtil.prepareTenant;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import io.restassured.http.Header;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import java.net.MalformedURLException;
+import java.util.List;
+import org.folio.rest.jaxrs.model.Physical;
+import org.folio.rest.jaxrs.model.PoLine;
+import org.folio.rest.jaxrs.model.PoLine.OrderFormat;
+import org.folio.rest.jaxrs.model.PoLineCollection;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SearchOrderLinesTest extends TestBase {
```

Comment:
Please add this test class to `StorageTestSuite` because at the moment it is ignored (please have a look at [MODORDSTOR-73 #2](https://jenkins-aws.indexdata.com/job/folio-org/job/mod-orders-storage/job/MODORDSTOR-73/2/testReport/org.folio.rest.impl/)

Reply:
Done